lib/classifier-reborn/backends/bayes_redis_backend.rb

```diff
@@ -33,7 +36,7 @@ def initialize(options = {})
         raise NoRedisError
       end
 
-      @redis = Redis.new(options)
+      @redis = options.fetch(:redis_conn, Redis.new(options))
```

Comment:
Minor point but my preference for the argument name would just be `redis`.
```suggestion
      @redis = options.fetch(:redis, Redis.new(options))
```